stories/Rating/Rating.stories.mdx

```diff
@@ -20,14 +21,11 @@ Per visualizzare una label utilizzare l'attributo `legend` del componente `Ratin
 <div className='docs'>
   <Callout color='success'>
     <CalloutText>
-      <p>
-        L'attributo <code>&lt;legend&gt;</code> contiene testo aggiuntivo per
-        Screen Reader all’interno di uno{' '}
-        <code>&lt;span class="sr-only"&gt;</code>.<br />
-        La label con il numero di stelle dev’essere contenuta in uno <code>
-          &lt;span&gt;
-        </code> semplice.
-      </p>
+      L'attributo <Code>&lt;legend&gt;</Code> contiene testo aggiuntivo per
+      Screen Reader all’interno di uno <Code>&lt;span class="sr-only"&gt;</Code>
+      .<br />
+      La label con il numero di stelle dev’essere contenuta in uno{' '}
```

Comment:
Need an explicit space here and below because of the `<Code>` tag in the new line. Could not find a better way to have an empty space (other than the `&nbsp;` which is even uglier) for this case.

Reply:
Alright!